cityguide/source/app/src/main/res/values/strings.xml

```diff
@@ -20,15 +19,21 @@
 <resources>
 
     <string name="app_name">City Guide Demo</string>
+
     <string name="action_set_location">Set location</string>
     <string name="select_area">Select area:</string>
     <string name="select_city">Select city:</string>
     <string name="areas_title">Areas</string>
     <string name="please_wait">Please wait...</string>
     <string name="show_on_map">Show on map</string>
+
+    <string name="no_areas">Kaa client is not started. Wait for it! We show your areas!</string>
```

Comment:
Areas info isn't available. The areas will be shown as soon as Kaa client will be started.